docs/nations/air/council.md

```diff
@@ -1,18 +1,19 @@
 ---
-title: Air Council
+title: Air Councils
 ---
 
 # Air Council System
 
 ## Overview of the Council System
 
-The Air Collective is divided into 3 locations:
+The Air Collective is divided into 4 locations:
 
+- Eastern Air Temple
 - Southern Air Temple
 - Western Air Temple
 - Gyantse Village
 
-Each one of them is independent with their own councils. Those councils are established by one prominent leader (Abbot, Abbess, Mayor) and are accompanied by two senior Monks or Nuns.
+Each location is independent and run by its own council. The councils consist of a leader who is an Abbot, Abbess or Mayor, and 2 senior Monks or Nuns.
 
 The people elect an Abbot, Abbess or Mayor through a vote. The elected leaders have a four-month grace period; after that, other political opponents may challenge them to an election. 
```

Comment:
How are the 2 Senior Monks/Nuns put into the council? Are they also elected or just chosen by the Abbot/Abbess/Mayor?

Reply:
As far as I know they're not elected, so chosen by the Abbot/Abbess/Mayor